components/entitlement/org.wso2.carbon.identity.entitlement.ui/src/main/resources/web/entitlement/authorization-index.jsp

```diff
@@ -239,23 +240,26 @@
             %>
             <tr>
                 <td style="width:100px;">
-                    <a class="icon-link" onclick="updownthis(this,'up',<%=request.getParameter("pageNumber")%>,<%=numberOfPages%>)" style="background-image:url(../admin/images/up-arrow.gif)"></a>
-                    <a class="icon-link" onclick="updownthis(this,'down',<%=request.getParameter("pageNumber")%>,<%=numberOfPages%>)" style="background-image:url(../admin/images/down-arrow.gif)"></a>
+                    <a class="icon-link" onclick="updownthis(this,'up',<%=Encode.forHtmlContent(request.getParameter("pageNumber"))%>,
+                            <%=numberOfPages%>)" style="background-image:url(../admin/images/up-arrow.gif)"></a>
+                    <a class="icon-link" onclick="updownthis(this,'down',<%=Encode.forHtmlContent(request.getParameter("pageNumber"))%>,
```

Comment:
We should encode for html content if the function updownthis injects the passed parameter to an html. Otherwise just encode for javascript attribute